modules/menu/media/css/about.css

```diff
@@ -7,12 +7,16 @@
 }
 
 .popup-about-table {
```

Comment:
It would seem that these changes should be made in the layout.css file instead.
Located at: /application/views/css/layout.css

I thought that file was generated when we bundle our js, but that does not seem to be the case.
Perhaps we did that some time ago, but it seems that the layout.css file has several direct commits so we probably shouldn't rebuild it.